qdmr_parsing/model/run_model.py

```diff
@@ -4,11 +4,22 @@
 import os
 import pandas as pd
 import random
+import sys
+from copy import deepcopy
 
 from evaluation.decomposition import Decomposition, get_decomposition_from_tokens
 from model.rule_based.rule_based_model import RuleBasedModel
 from model.rule_based.copy_model import CopyModel
 from model.seq2seq.seq2seq_model import Seq2seqModel
+from utils.preprocess_examples import fix_references
+
+# sys.path.append('..') below a workaround to solve the small dependency between the root
+# directories "qdmr_parsing" and "annotation_pipeline", which have been developed separately 
+# but are under the same GitHub repository.'''
+
```

Comment:
please remove the extra line break (the comment should be just above the relevant lines of code